packages/web-components/src/components/content-item-horizontal/content-item-horizontal-thumbnail-copy.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, customElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import DDSContentItemCopy from '../content-item/content-item-copy';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import styles from './content-item-horizontal-thumbnail.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * The copy content of content item horizontal with thumbnail.
+ *
+ * @element dds-content-item-horizontal-thumbnail-copy
+ */
+@customElement(`${ddsPrefix}-content-item-horizontal-thumbnail-copy`)
+class DDSContentItemHorizontalThumbnailCopy extends StableSelectorMixin(DDSContentItemCopy) {
```

Comment:
I don't think this component is needed as the `dds-content-item-horizontal-copy` has already existed and can be used within the variant of `content-item-horizontal` layout component with thumbnail. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/content-item-horizontal/content-item-horizontal-copy.ts

Similar comment as noted here - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/6425/files#diff-962566a3c5ebb5bad8f6364e5cdadebedeb244446091770dc9b678a89b3f2743R25